packages/preview/university-assignment/0.1.0/typst.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "university-assignment"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Alessandro Mason <alessandro.mason@example.com>"]
+license = "MIT"
+description = "A flexible template for university assignments, labs, and homework with customizable details"
+repository = "https://github.com/alessandromason/typst-university-assignment"
+keywords = ["template", "university", "assignment", "homework", "lab"]
+categories = ["layout"]
+disciplines = ["computer-science", "engineering"]
+
+exclude = ["example.typ"]
```

Comment:
If you want to make this a template from which you can bootstrap a new project, should not exclude this file, move it to a new sub-directory and fill the fields of the `[template]` section in your manifest.